fontc_crater/src/ttx_diff_runner.rs

```diff
@@ -253,8 +253,21 @@ pub(super) fn print_report(results: &Results<DiffOutput, DiffError>, verbose: bo
     }
 }
 
+fn assert_has_timeout_coreutil() {
+    match Command::new("which").arg("timeout").output() {
+        Ok(out) if out.status.success() => return,
+        Err(e) => eprintln!("'which' failed!? '{e}'"),
+        Ok(_) => eprintln!("could not find 'timeout'. You may need to install coreutils"),
```

Comment:
Suggest giving the exact command and indicating the advice is mac-specific

Reply:
sorry, exact command where? I thought the 'which' branch was academic since I thought `which` was in the POSIX spec, but apparently it's not, and just a shell builtin? but still hard to imagine hitting that branch.

I didn't provide mac specific advice since in theory this could also be hit on windows (or various other places that are not GNU/Linux)

Reply:
I mean to install coreutils on a mac. I don't think anything but a mac is likely to be missing timeout.

I hadn't considered windows, I think deeming windows out of scope for crater is completely fine.